### PR TITLE
Configuration of docker-compose.prod.yml updated. build.sh changed.

### PR DESCRIPTION
**Pull request by Netmisa**
# Description

In this PR we download images directly from custom registry (temporary)

# How to test this ?

```
./docker/build.sh
./docker/build_dev.sh
```

### CODE REVIEW

File: docker/php/Dockerfile [hunk 1]
```diff
@@ -30,9 +30,10 @@ RUN BUILD_DEPS="g++ libicu-dev" && \
 RUN echo 'date.timezone = "UTC"' >> /usr/local/etc/php/php.ini
 RUN echo 'opcache.max_accelerated_files = 20000' >> /usr/local/etc/php/conf.d/docker-php-ext-opcache.ini
 
-COPY docker/php/php-fpm.d/hermod.conf /usr/local/etc/php-fpm.d/hermod.conf
+# COPY docker/php/php-fpm.d/hermod.conf /usr/local/etc/php-fpm.d/www.conf
```
Comment (vincentlepot) [inline docker/php/Dockerfile]: No need to keep commented code ;)

File: docker/php/run.sh [hunk 2]
```diff
@@ -6,5 +6,7 @@
 ./bin/console cache:clear --env prod
 ./bin/console cache:warmup --env prod
 
+./docker/wait-for-it.sh -t 0 ${POSTGRES_HOST}:${POSTGRES_PORT} && \
```
Comment (vincentlepot) [inline docker/php/run.sh]: If you copy wait-for-it in / why do use the one in ./docker?

File: docker/php/Dockerfile (hunk 1 above)
Comment (Netmisa) [inline docker/php/Dockerfile]: My bad, Thanks

File: docker/php/run.sh (hunk 2 above)
Comment (Netmisa) [inline docker/php/run.sh]: I will remove other `COPY` because docker folder will be pushed.

**Pull request merged**